AlTextAdventure/Program.cs

```diff
@@ -16,28 +19,52 @@ static void Main(string[] args)
                 Console.WriteLine("Great! Would you like a drink?");
                 response = Console.ReadLine();
 
-           
                 if (response == "Yes")

```

Comment:
This is good and works, but consider doing something like `if (response.ToUpper() == "YES")` or `if (response.ToLower() == "yes")` so that capitalization isn't a concern